lib/Word_Lib/Word_Syntax.thy

```diff
@@ -18,22 +18,22 @@ context
 begin
 
 abbreviation
-  wordNOT  :: "'a::len word \<Rightarrow> 'a word"      ("~~ _" [70] 71)
+  wordNOT  :: "'a::len word \<Rightarrow> 'a word"      (\<open>(\<open>open_block notation=\<open>prefix ~~\<close>\<close>~~ _)\<close> [70] 71)
```

Comment:
what are these strange complicated notation blocks doing? is this even documented?

Reply:
Yes, it's document in NEWS (and the reference manual, I presume, but I have not checked). I don't actually fully understand what this is doing yet, but I haven't really looked into it. Something with having blocks that do not affect pretty printing.